CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+### 3.4.1
```

Comment:
New Feature - so either 3.5.0 - or as part of 3.4.0 (if we can get this PR merged today so that we can cut 3.4.0 release for both new features). Let's discuss in today's sync-up.

Reply:
3.4.0 sounds good to me and can try to get it merged today.